src/helics/core/BrokerBase.cpp

```diff
@@ -108,11 +108,14 @@ std::shared_ptr<helicsCLI11App> BrokerBase::generateBaseCLI ()
 {
     auto hApp = std::make_shared<helicsCLI11App> ("Arguments applying to all Brokers and Cores");
 
-    hApp->add_option ("--federates,-f,--min_federates,--minfed,-m", minFederateCount,
-                      "the minimum number of federates that will be connecting");
+    hApp
+      ->add_option ("--federates,-f,--minfederates,--minfed,-m", minFederateCount,
+                    "the minimum number of federates that will be connecting")
+      ->ignore_underscore ();
     hApp->add_option ("--name,-n,--identifier", identifier, "the name of the broker/core");
-    hApp->add_option ("--maxiter,--max_iterations", maxIterationCount, "the maximum number of iterations allowed",
-                      true);
+    hApp->add_option ("--maxiter,--max_iterations", maxIterationCount, "the maximum number of iterations allowed")
```

Comment:
Is `max_iterations` one of the arguments we want to remove the underscore from?

Reply:
probably, I will change it in the next commit